app/views/resources/index.html.erb

```diff
@@ -1,37 +1,88 @@
 <%= render "layouts/navbar", page: "Resources" %>
 
-<div class="container">
-    <h1 class="ui-shescoding-crud">Resources</h1>
-    <div class="col-sm-12">
-        <div class="row">
-            <div class="ui-shecodes-mission">
-              <table>
-                <thead>
-                  <tr>
-                    <th>Title</th>
-                    <th>Description</th>
-                    <th>Tags</th>
-                    <th colspan="3"></th>
-                  </tr>
-                </thead>
-
-                <tbody>
-                  <% @resources.each do |resource| %>
-                    <tr>
-                      <td><%= resource.title %> |</td>
-                      <td>| <%= resource.description %></td>
-                      <td>| <%= raw resource.tag_list.map { |t| link_to t, tag_path(t) }.join(', ') %></td>
-                      <td>| <%= link_to 'Show', resource %></td>
-                      <td>| <%= link_to 'Edit', edit_resource_path(resource) %></td>
-                      <td>| <%= link_to 'Destroy', resource, method: :delete, data: { confirm: 'Are you sure?' } %></td>
-                    </tr>
-                  <% end %>
-                </tbody>
-              </table>
-
-              <p><%= link_to 'New Resource', new_resource_path %></p>
-
+<div class="resources-section container">
+  <div class="row">
+    <div class="resources-sidebar col-md-3">
+      <ul>
+        <li class="selected" data-category="*"><%= link_to "All Resources", "" %></li>
```

Comment:
Hey @natsteinmetz,

The library I used for the category filtering overrides the link and defaults to filtering the resources on the page. What would the preferred behavior be?

When you click on any of the categories from a tag page it goes back to the main resource page and shows all items in that category? Or should it stick to filtering the items under the tag and let the user navigate back with the breadcrumbs?